src/app/find-help/view-offer-accommodation/view-offer-accommodation.component.html

```diff
@@ -1,7 +1,7 @@
 <app-breadcrumb></app-breadcrumb>
 
 <div class="d-flex flex-column flex-md-row" *ngIf="data">
-  <mat-card class="offer p-4 w-100 w-md-50">
+  <mat-card class="offer p-4 w-100 w-md-50 d-flex flex-column">
```

Comment:
Generally approved and is good, if you have time I still wonder if d-flex is needed here at all.
you could remove d-flex and flex-column as divs are displeyed like flex-column by default

then instead of flex-grow-1 would be w-100 so 2 classes removed then ;)

Reply:
I don't understand :) If add `w-100` instead of `flex-grow-1` and remove `d-flex flex-column`. Same view
![image](https://user-images.githubusercontent.com/43149504/161743157-0c1d46f0-3e3c-4a5b-8fcc-125809d2104b.png)

Reply:
Still not sure if I understand you correctly

Reply:
Without these classes back link is not pinned to the bottom (as on the screenshot above)

Reply:
Ok i have checkd this. It is not working then.